ilp-core/src/main/java/org/interledger/core/InterledgerProtocolException.java

```diff
@@ -27,4 +27,10 @@ public InterledgerProtocolException(final InterledgerRejectPacket interledgerRej
   public InterledgerRejectPacket getInterledgerRejectPacket() {
     return interledgerRejectPacket;
   }
+
+  @Override
+  public String toString(){
+      return interledgerRejectPacket.toString();
+  }
+
 }
```

Comment:
As discussed in #89 I think this should include the value of `this.getMessage()`